README.md

```diff
@@ -0,0 +1,2 @@
+# thegooddocsproject.github.io
+Forking The Good Docs Project website
```

Comment:
As you are doing a lift-and-shift, and this file doesn't contain any value. I'd suggest that we don't include this README.md file. (We can add it later.)

Reply:
Noted and readme.md deleted.